### PR TITLE
Domain change: Domain purchase success screen

### LINKED ISSUE #8366: Create a new screen with the domain name, and a CTA to go back to the domain dashboard screen

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #8366, a subtask of #8238.

The PR just adds the domain purchase success screen, which displays the selected domain. There was a problem with exporting a vector image from Figma (I asked Ann if she could fix it) so in the meantime I used a PNG as a placeholder.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/domain/PurchaseSuccessfulScreen.kt [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    viewModel.viewState.observeAsState().value?.let { state ->
+        Crossfade(targetState = state) { viewState ->
+            PurchaseSuccessful(viewState, viewModel::onDoneButtonClicked)
+        }
+    }
+}
+
+@Composable
+private fun PurchaseSuccessful(viewState: ViewState, onDoneButtonClicked: () -> Unit) {
+    Column(modifier = Modifier.background(MaterialTheme.colors.surface)) {
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/domain/PurchaseSuccessfulScreen.kt]: Np, I'd make the content scrollable. At least the part above the "Done" button. If you rotate the screen you can't see anything.

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Looks good @0nko, nice one! I left a minor non blocking np. 

On other side of things I was wondering if we should provide a different illustration for dark mode as we do on other screens. I think the current one looks good in dark mode though (once we update it to vector). 
cc @annchichi 

 

Btw friendly reminder (in case you haven't addressed it already), we should update the color of the "Primary" domain tag for dark mode:

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/domain/PurchaseSuccessfulScreen.kt (hunk 1 above)
Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/domain/PurchaseSuccessfulScreen.kt]: Good call! 👍

**Comment by 0nko** [comment]
Thank you for the review, @JorgeMucientes. I've addressed your comments, including the primary domain tag:

image

I've left the illustration as is since we don't have a dark variant and I also think it looks good.

**Pull request merged**